src/LibraryViewExtension/LibraryViewController.cs

```diff
@@ -191,11 +181,17 @@ internal LibraryView AddLibraryView()
             this.browser = browser;
             sidebarGrid.Children.Add(view);
             browser.RegisterAsyncJsObject("controller", this);
-            //RegisterResources(browser);
+            RegisterResources(browser);
 
             view.Loaded += OnLibraryViewLoaded;
             browser.SizeChanged += Browser_SizeChanged;
             browser.LoadError += Browser_LoadError;
+
+            // TODO - this was added Fall 2017 due to issues with 
+            // library failing to load due to timing issues.  DYN-944 
+            // is a testing task that should make the final determination
+            // as it no longer seems required in CEF v65.0.1
+            /*
```

Comment:
This is my biggest concern, without registering our resources immediately instead of using the commented out function below the librarie fails to load.  I have simply restored resource handling to it's previous state which seems to do the trick but we need to be on the look out for timing related issues in the testing task.  See [DYN-944](https://jira.autodesk.com/browse/DYN-944) for additional details.